### DISCUSSION TITLE
Error in migrate no MySQL

### DISCUSSION THREAD

**Discussion by altendorfme** [dani-garcia/vaultwarden#1899] [Q&A]
I'm having trouble migrating the sqlite database to MySQL, I followed the documentation process:

`docker run -d --name vaultwarden -e RUST_BACKTRACE=1 -e DATABASE_URL='mysql://vaultwarden:PASSWORD@127.0.0.1:3306/vaultwarden' -e ADMIN_TOKEN=XYZ -e ENABLE_DB_WAL='false' -v /docker/vaultwarden/:/data/ -p 80:80 vaultwarden/server:latest`

`docker stop vaultwarden`

`sqlite3 db.sqlite3 .dump | grep "^INSERT INTO" | grep -v "__diesel_schema_migrations" > sqlitedump.sql ; echo -ne "SET FOREIGN_KEY_CHECKS=0;\n$(cat sqlitedump.sql)" > mysqldump.sql`

`mysql --force --password --user=vaultwarden --database=vaultwarden < mysqldump.sql`

```
ERROR 1146 (42S02) at line 2: Table 'vaultwarden.users' doesn't exist
ERROR 1146 (42S02) at line 3: Table 'vaultwarden.devices' doesn't exist
...
ERROR 1146 (42S02) at line 10: Table 'vaultwarden.folders_ciphers' doesn't exist
...
ERROR 1146 (42S02) at line 136: Table 'vaultwarden.ciphers' doesn't exist
...
```

If i try fix:
`sed -i 's#\"#\#g' mysqldump.sql`
```
sed: -e expression #1, char 8: unterminated `s' command
```

----

`mysql --version`
```
mysql  Ver 15.1 Distrib 10.3.29-MariaDB, for debian-linux-gnu (x86_64) using readline 5.2
```

**Comment by BlackDex** [comment]
When you run the mysql connection of vaultwarden, do you see the tables being created? And if you check the database it self, do you see those tables your self?

**Reply by altendorfme** [reply]
No table is created in the database and no error message appears, not even in the log.

**Reply by altendorfme** [reply]
I found the problem, it was in the database hostname, I was using localhost and I ignored the docker issue.
